lapack.go

```diff
@@ -29,4 +29,35 @@ type Float64 interface {
 	Dpotrf(ul blas.Uplo, n int, a []float64, lda int) (ok bool)
 }
 
-type Complex128 interface{}
```

Comment:
It seems we need to keep this - just with documentation saying that it is the set of LAPACK functions we support.